web/frontend/src/components/remote-control-cards.vue

```diff
@@ -269,8 +269,8 @@ const restartStatusStream = () => {
       updateStatus((response).getStatusList());
       lastStatusTS = Date.now();
     });
-    statusStream.on('status', (newStatus: { details: unknown }) => {
-      if (!ConnectionClosedError.isError(newStatus.details)) {
+    statusStream.on('status', (newStatus?: { details: unknown }) => {
+      if (!ConnectionClosedError.isError(newStatus!.details)) {
```

Comment:
If this is now an optional then why are we doing a non-null assertion?

Reply:
I am trying to preserve the old behavior and fail fast

Reply:
@nicksanford how do you feel about us making this failure more graceful? i.e. does an `undefined` robot streaming status warrant console errors or can we just return?

Reply:
What does an `undefined` robot streaming status mean?

Reply:
I don't know unfortunately - this typing change was a bit of a surprise to me to be honest, which is why I want to keep an assertion for now. @micheal-parks is this okay or do you feel strongly that we shouldn't allow these sorts of errors in RC?

Reply:
Can you please add me as a reviewer on the PR that is not closed to make this change? I'd like confirmation that:
1. successfully assembling the map from the GetPointCloudMap stream works
2. if there is an error in returning the stream there is an error logged & shown to the user
3. its not possible for the stream to terminate w/o this code calling either `resolve` or `reject`

Can you please test those conditions are true before merging or pair with me if it is not obvious how to do so?